common-lib/src/main/scala/com/gu/mediaservice/lib/auth/Authentication.scala

```diff
@@ -63,15 +66,19 @@ class Authentication(config: CommonConfig, actorSystem: ActorSystem,
   }
 
   final override def validateUser(authedUser: AuthenticatedUser): Boolean = {
-    authedUser.user.email.endsWith("@guardian.co.uk") && authedUser.multiFactor
+    val isValidDomain = authedUser.user.email.endsWith("@" + userValidationEmailDomain)
+    val passesMultifactor = if(multifactorChecker.isEmpty) { authedUser.multiFactor } else { true }
```

Comment:
Is the multifactorChecker.isEmpty boolean inverted?
ie. if there is a MFA checker, then check if the user has MFA else pass them.
Does this currently read is there is no MFA checker then check their MFA?

Reply:
Yes, great spot! Thanks @tonytw1, I will add this to the long list of times I've used the wrong `isEmpty/nonEmpty` and also write a test